ddtrace/contrib/grpc/aio_client_interceptor.py

```diff
@@ -160,13 +152,9 @@ async def _wrap_stream_response(
         span: Span,
     ) -> ResponseIterableType:
         try:
+            _handle_add_callback(call, _done_callback(span))      
```

Comment:
iiuc this change would mean that in the successful iterable response case, the span would not include the status code and details. I would expect us to have a test case that covers that behavior, which we would be breaking. Can we avoid this change to the generated span?

Reply:
I do not think so, after the last response is yielded when iterating the generator, the code after the loop is never executed in our stream wrapper. That means, we can't call the async functions of `code()` and `details()`. 

Our only option is adding the callback before looping through the response generator, but we can't add code and details since they are async and the callback is sync. 

As far as i can tell, there are no streaming spans currently being created (for async client interceptors atleast) within **ALL** of Org 2, so this should not be a breaking change.

Reply:
yes, updated